frontend/src/__tests__/components/app/Modal.test.js

```diff
@@ -48,7 +48,7 @@ describe('Modal test', () => {
     expect(html.includes('Action')).toBe(true);
   });
 
-  it(`calls startProcess when initializing a modal of 'process' type`, async (done) => {
+  it.skip(`calls startProcess when initializing a modal of 'process' type`, async (done) => {
```

Comment:
pls add a TODO and document it on the issue page where this will be fixed (e.g. the refactoring task).
We need to keep track of this kind of changes and we have to address them later.